src/config.rs

```diff
@@ -23,6 +23,9 @@ pub struct Config {
 
     #[serde(default = "default_write_remote_cache")]
     pub write_remote_cache: bool,
+
+    #[serde(default = "default_docker_cli")]
+    pub docker_cli: String,
```

Comment:
I think this would belong more naturally at the top, before `docker_repo`. That way, the `docker_` fields are together. What do you think? (If you agree, please change the order here and in all the relevant places below.)

Reply:
Agreed. Some projects prefer additions at the end, so I wasn't sure which was the case here. I'll make those changes.